cli/stork-publisher-agent/README.md

```diff
@@ -0,0 +1,63 @@
+# Stork Publisher Agent
+
+The easiest way to become a Stork Publisher is to run the Stork Publisher Agent docker container on your infrastructure and send price updates to the Agent through a local websocket. The Stork Publisher Agent will sign your price updates with your private key and send them to the Stork Network.
```

Comment:
Do we want to say anything about the data being auto-routed?

Reply:
I tried to abstract over that by saying we "send updates to the Stork Network" - not sure if publishers will care about the routing logic. Happy to add it if you think it's useful though